### ISSUE TITLE
Segmentation fault on reading parquet file

### ISSUE THREAD

**Issue by k-anshul** [duckdb/duckdb#5968]
### What happens?

Duckdb cli crashes with segmentation fault on trying to read the following parquet file : https://github.com/k-anshul/pqarrow_example/blob/main/out.parquet

The parquet file seems valid since I can scan it with other cli based tools :  https://github.com/apache/parquet-mr/blob/master/parquet-cli/README.md

### To Reproduce

1. Download given file in local 
2.  select * from read_parquet("filename");
3. duckdb will crash with segmentation fault

### OS:

mac os 13.1 (m1)

### DuckDB Version:

v0.6.1

### DuckDB Client:

cli

### Full Name:

Anshul

### Affiliation:

Independent Consultant

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Thanks for the report! I have pushed a fix in #5969.

I do think the file is incorrectly written - however. The problem is that the root schema is labeled with a `REPEATED` repetition type. In the parquet spec it says [the following](https://github.com/apache/parquet-format/blob/bfc549b93e6927cb1fc425466e4084f76edc6d22/src/main/thrift/parquet.thrift#L366):

```
  /** repetition of the field. The root of the schema does not have a repetition_type.
   * All other nodes must have one */
3: optional FieldRepetitionType repetition_type;
```

I have gotten this file to work by just ignoring the repetition type of the root element - even if it is set. I suspect this is what other readers do as well.

**Comment by k-anshul** [comment]
Thank you for the prompt response. 
I had used [pqarrow](https://github.com/apache/arrow/tree/master/go/parquet/pqarrow) to generate the parquet file.
As you correctly mentioned it was using root's repetition type as REPEATED by default. I changed it to REQUIRED and it started working with older duck db impl as well.
